specification/dartLangSpec.tex

```diff
@@ -4720,7 +4789,7 @@ \section{Interfaces}
 such that we can say that an interface 'declares' or 'has' a member,
```

Comment:
Why are these red in the review tool?

Reply:
I think it considers the use of a single quote at both ends of a phrase as a syntax error, in the sense that it should have been a backquote and a single quote. Fixed approx. 50 locations where this was done inconsistently. There will still be a few red spots, because github highlighting doesn't understand all situations as intended.